src/openforms/middleware.py

```diff
@@ -18,3 +20,18 @@ def __call__(self, request):
             value = SAMESITE_VALUE if origin and not settings.DEBUG else "Lax"
             response.cookies[settings.SESSION_COOKIE_NAME]["samesite"] = value
         return response
+
+
+class SessionTimeoutMiddleware:
+    """
+    Allows us to set the expiry time of the session based on what
+    is configured in our GlobalConfiguration
+    """
+
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
```

Comment:
This call should ALSO handle form_session_timeout. I think something like this can solve it:

```
if request.user.is_staff:
  # set expiry(admin timeout)
else:
  # set expiry(user timeout)
```